app/views/recipes/show.html.erb

```diff
@@ -1,4 +1,8 @@
-<%= render @recipe %>
+<% if can? :read, @recipe %>
+  <%= render @recipe %>
+<% else %>
+  <p>Sorry! You do not have the permission to view this recipe.</p>
+<% end %>
 
```

Comment:
This view is almost complete! 🍾 

> Keep it up! you are almost done!

Please modify this view so it has the wireframe layout:
- Update the `recipes#show` so it displays a table with:
  - A column with the food item's name.
  - A column with the food_recipes item's quantity.
  - A column with the total value for the food_recipes item. (In the `food_recipe` model make a method that calcules it) 
  - A column with a button that removes the food_recipes item from `food_recipes` table.
  
  Ex: 
```
<table class="table table-striped">
    <thead>
      <tr>
        <th>Food</th>
        <th>Quantity</th>
        <th>Value</th>
        <th>Actions</th>
      </tr>
    </thead>
    <% if @food_recipes.any? %>
      <tbody>
      <% @food_recipes.each do |food_recipe| %>
        <tr>
          <td><%= food_recipe.food.name %></td>
          <td><%= food_recipe.quantity_with_measurement %></td>
          <td><%= food_recipe.total_value %></td>
          <td>
            <%= link_to "Remove", food_recipe_path(food_recipe.id), class: "btn btn-ountline-danger", method: :delete, data: { turbo_method: :delete } %>
          </td>
        </tr>
        <% end %>
      </tbody>
      <% else %>
        <tr>
          <td colspan="3">Empty list for now</td>
        </tr>
      <% end %>
  </table>
```